test/data/threads.ts

```diff
@@ -269,6 +271,27 @@ export const CREATE_GORE_THREAD_RESPONSE: Thread = {
   },
 };
 
+// todo: finish filling fields
+export const DELETED_THREAD_SUMMARY: ThreadSummary = {
```

Comment:
I don't think you need this. Crossing fingers you should just be able to use any thread that already exists and the test DB setup will automatically rollback your query at the end of your tests (but let me know if this doesn't happen, cause that logic is a bit finicky sometimes). Mostly, adding a new thread will cause *a lot* of tests to fail cause they rely on how many posts we have in the DB.

(Yes, there's some bad practices at play here....)

I'm also fine with keeping it, but you'll have to fix the tests